NuGet.Config

```diff
@@ -3,5 +3,6 @@
   <packageSources>
     <add key="nugetv3" value="https://api.nuget.org/v3/index.json" />
     <add key="myget" value="https://www.myget.org/F/azureiiot/api/v3/index.json" />
+    <add key="opcua-preview" value="https://opcfoundation.pkgs.visualstudio.com/opcua-netstandard/_packaging/opcua-preview/nuget/v3/index.json" />
```

Comment:
How do we make sure we don't accidentally pull in preview packages? Are they marked "-preview"?

Reply:
yep, they are all marked -preview